modules/node_modules/@ncigdc/containers/FrequentMutationsTable.js

```diff
@@ -0,0 +1,364 @@
+  data.map((hit) => {
+    const { transcript } = hit.consequence.hits.edges.find(c => c.node.transcript.is_canonical).node;
+    const { annotation = {}, consequence_type = '', gene_symbol, aa_change } = transcript;
+    const impact = annotation.impact;
+
+    return {
+      ...hit,
+      score: hit.score,
+      num_affected_cases_all: hit.occurrence.hits.total,
+      num_affected_cases_project: hit.occurrence.hits.edges.filter(o =>
+        o.node.case.project.project_id === projectId
+      ).length,
+      num_affected_cases_by_project: hit.occurrence.hits.edges.reduce((acc, o) => ({
+        ...acc,
+        [o.node.case.project.project_id]: acc[o.node.case.project.project_id] ? acc[o.node.case.project.project_id] + 1 : 1,
+      }), {}),
+      impact,
+      consequence_type: (
+        <span>
+          <b>{startCase(consequence_type.replace('variant', ''))}</b>
+          <span style={{ marginLeft: '5px' }}>
+            <GeneLink id={gene_symbol}>
+              {gene_symbol}
+            </GeneLink>
+          </span>
+          <span
+            style={{
+              marginLeft: '5px',
+              color: impactColors[impact] || 'inherit',
+            }}
+          >
+            {truncate(aa_change)}
+          </span>
+        </span>
+      ),
+    };
+  })
+);
+
+const FrequentMutationsTableComponent = compose(
+  withRouter,
+  lifecycle({
+    componentDidMount() {
+      this.props.relay.setVariables({
+        fetchData: true,
+        fmTable_offset: parseIntParam(this.props.query.fmTable_offset, 0),
+        fmTable_size: parseIntParam(this.props.query.fmTable_size, 20),
+        fmTable_filters: parseFilterParam(this.props.query.fmTable_filters, this.props.defaultFilters || null),
+      });
+    },
+    componentWillReceiveProps(nextProps) {
+      if (!isEqual(this.props.query, nextProps.query)) {
+        this.props.relay.setVariables({
+          fmTable_offset: parseIntParam(nextProps.query.fmTable_offset, 0),
+          fmTable_size: parseIntParam(nextProps.query.fmTable_size, 20),
+          fmTable_filters: parseFilterParam(nextProps.query.fmTable_filters, nextProps.defaultFilters || null),
+        })
+      }
+    }
+  }),
+  withTheme,
+  withSize()
+)(({
+  numCasesAggByProject = {},
+  projectId = '',
+  showSurvivalPlot = false,
+  selectedSurvivalData = {},
+  setSelectedSurvivalData = () => {},
+  cohort: { ssms, cases },
+  relay,
+} = {}) => {
+  if (!ssms) {
+    return (
+      <Row style={{ justifyContent: 'center', padding: '1rem' }}>
+        <SpinnerParticle />
+      </Row>
+    );
+  }
+  const frequentMutations = mapData(ssms.hits.edges.map(x => x.node), projectId);
+  return (
+    <div>
+      <Row style={{ backgroundColor: 'white', padding: '1rem', justifyContent: 'space-between' }}>
+        <Showing
+          docType="mutations"
+          prefix={'fmTable'}
+          params={relay.route.params}
+          total={ssms.hits.total}
```

Comment:
A little bit of weirdness with this 
![image](https://cloud.githubusercontent.com/assets/3953093/23675603/d5c198d8-0347-11e7-98f4-40231ac2bd2e.png)
I think it just needs to use `parseInt` for the size/offset inside Showing.